### DISCUSSION TITLE
Date-time picker on value metric

### DISCUSSION THREAD

**Discussion by DranzerHMS** [laravel/nova-issues#6102] [Ideas & Feature Requests]
Hi hope all is well, 

I have a request for the Dashboard metrics - I think a date-time picker on the metric would be a nice addition for the user to pick specific dates to filter down the results between these dates instead of using the other date ranges or maybe even allow to use both somehow but having a date-time picker would be useful

Thank you appreciate any help

**Comment by proilyxa** [comment]
yes, we need to this feature
